lms/templates/index_overlay.html

```diff
@@ -35,7 +35,7 @@ <h1>How we will support you</h1>
         <div class="row gx-2">
             <div class="col">
                 <div class="p-3">
-                    <img class="img-responsive" src="https://d2l03dhf2zcc6i.cloudfront.net/images/networking.jpeg"
+                    <img class="img" src="https://d2l03dhf2zcc6i.cloudfront.net/images/networking.jpeg"
```

Comment:
any reason this was changed?

Reply:
the class was making the two images not stack when on a smaller screen. The rest of the images don't have this class